### PR TITLE
Initial Commit for Database

### PR DESCRIPTION
**Pull request by Xbey25**
Creation and generation of migration and tables. Placing data into the database works. Currently, the data is working if you have some preexistent data since the conditions for some of the table values are NOT NULL.

### CODE REVIEW

File: C#/Domain/Models/Patient.cs [hunk 1]
```diff
@@ -4,10 +4,13 @@ public class Patient
 {
     public string Name { get; set; }
 
+    public int Id;
```
Comment (Eli3336) [inline C#/Domain/Models/Patient.cs:7]: Could u put the {get;set;} and don't you need the [Key] for database? I can see that there is only one key in SensorValue, but not anywhere else and I am a bit confused since the last time I saw the code on disc there was this attribute to them.

File: C#/EfcDataAccess/SensorEfcDao.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        await context.SaveChangesAsync();
+        return newSensor.Entity;
+    }
+
+    public async Task<Sensor?> GetById(int id)
+    {
+        Sensor? sensor = await context.Sensors.FindAsync(id);
+        return sensor;    }
+
+    public async Task<IEnumerable<SensorValue>> GetSensorsValuesAsync(int? roomId)
```
Comment (Eli3336) [inline C#/EfcDataAccess/SensorEfcDao.cs:31]: Is this method for the log or the current sensor value? The IEnumerable lets me believe that it's the log, but then there is no method to get the current sensor value...and we have an endpoint for that, therefore I need that method.
And to get the log of a sensor and the current value, we need the sensor's id. We can't just get from every sensor without knowing which sensor we are taking the values of.
Basically, I need you to add the sensorId and make a separate method to get only one SensorValue (the last one that was saved in the database / the current one).

**Review by Eli3336 — CHANGES_REQUESTED** [review]
Please look at the comments and answer my question about the key(as I am confused :))).

File: C#/Domain/Models/Patient.cs (hunk 1 above)
Comment (Xbey25) [reply C#/Domain/Models/Patient.cs:7]: In the HospitalContext class, we designate the primary keys by the HasKey method. So this is not necessary, I will remove the attribute [Key].

I am unsure about the foreign keys if they are automatically generated from the relationship by having a List of values fx.

**Review by Eli3336 — CHANGES_REQUESTED** [review]

**Review by Eli3336 — APPROVED** [review]
